api/tests/mocha/services/users.spec.js

```diff
@@ -1341,6 +1344,121 @@ describe('Users service', () => {
         } ]);
       });
     });
+
+    it('should update the admin password in CouchDB config and not in user docs', async () => {
+      const data = { password: COMPLEX_PASSWORD };
+      couchSettings.getCouchConfig.resolves({
+        admin1: 'password_1',
+        admin2: 'password_2',
+      });
+      service.__set__('validateUser', sinon.stub().resolves({}));
+      service.__set__('validateUserSettings', sinon.stub().resolves({}));
+      sinon.stub(db.medic, 'put').resolves({});
+      sinon.stub(db.users, 'put').resolves({});
+
+      await service.updateUser('admin2', data, true);
+
+      chai.expect(db.medic.put.callCount).to.equal(1);
+      chai.expect(db.medic.put.args[0][0]).to.deep.equal({
+        _id: 'org.couchdb.user:admin2',
+        name: 'admin2',
+        type: 'user-settings',
+      });
+      chai.expect(db.users.put.callCount).to.equal(1);
+      chai.expect(db.users.put.args[0][0]).to.deep.equal({
+        _id: 'org.couchdb.user:admin2',
+        name: 'admin2',
+        type: 'user',
+      });
+      chai.expect(couchSettings.updateAdminPassword.calledOnce).to.be.true;
+      chai.expect(couchSettings.updateAdminPassword.args[0]).to.have.members(['admin2', COMPLEX_PASSWORD]);
```

Comment:
`to.have.members` is a bit risky to use for parameters, because it doesn't enforce order, when parameter order is important :) Please change this to `deep.equal`.

Reply:
Done